src/pages/dmt5/_margin_calculator.js

```diff
@@ -0,0 +1,292 @@
+import React, { useState, useEffect } from 'react'
+import Proptypes from 'prop-types'
+import { graphql, useStaticQuery } from 'gatsby'
+import styled from 'styled-components'
+import device, { size } from 'themes/device'
+import { Box, Flex, SectionContainer } from 'components/containers'
+import { Carousel, Header, LinkText, QueryImage, Text } from 'components/elements'
+import { LinkButton } from 'components/form'
+import { Localize } from 'components/localization'
+import { isBrowser } from 'common/utility'
+
+const query = graphql`
+    query {
+        margin_calculator: file(relativePath: { eq: "dmt5-margin-calculator.png" }) {
+            ...fadeIn
+        }
+        margin_calculator_mobile: file(relativePath: { eq: "dmt5-margin-calculator-mobile.png" }) {
+            ...fadeIn
+        }
+        swap_calculator: file(relativePath: { eq: "dmt5-swap-calculator.png" }) {
+            ...fadeIn
+        }
+        swap_calculator_mobile: file(relativePath: { eq: "dmt5-swap-calculator-mobile.png" }) {
+            ...fadeIn
+        }
+    }
+`
+
+const ImageWrapper = styled.div`
+    max-width: 39.2rem;
+    width: 100%;
+    height: 38.6rem;
+    object-fit: contain;
+    margin-bottom: 2.4rem;
+
+    @media ${device.mobileL} {
+        max-width: 232px;
+        width: 100%;
+        margin: 0 0 24px;
+        height: unset;
+
+        div {
+            max-width: 232px;
+            width: 100%;
+        }
+    }
+`
+
+const MainHeader = styled(Header)`
+    font-size: 48px;
+    margin: 0 0 12px;
+    line-height: 1.25;
```

Comment:
Will do ~ Though crowdin branch is currently missing the typography refactor. I'll just add a comment so that someone will refactor this once it is pushed to master.